birdhouse/default.env

```diff
@@ -117,6 +116,21 @@ __DEFAULT__CATALOG_PASSWORD="qwerty"
 __DEFAULT__PHOENIX_PASSWORD="phoenix_pass"
 __DEFAULT__PHOENIX_PASSWORD_HASH="sha256:123456789012:1234567890123456789012345678901234567890123456789012345678901234"
 
+BACKWARDS_COMPATIBLE_VARIABLES_PAVICS="
```

Comment:
> BIRDHOUSE_BACKWARD_COMPATIBLE_PAVICS_ALLOWED

I'm not sure I understand what would go in this variable

> Options/flags that modify how the compose script behaves should all use BIRDHOUSE_ as prefix

Yes! I love this idea.

Reply:
> Options/flags that modify how the compose script behaves should all use BIRDHOUSE_ as prefix

@fmigneault just to make sure we're on the same page here... we're talking about the following type of variable renaming right?:

- `DATA_PERSIST_ROOT` -> `BIRDHOUSE_DATA_PERSIST_ROOT`
- `SSL_CERTIFICATE` -> `BIRDHOUSE_SSL_CERTIFICATE`
- `DOC_URL` -> `BIRDHOUSE_DOC_URL`
- `MAGPIE_ADMIN_USERNAME -> `BIRDHOUSE_MAGPIE_ADMIN_USERNAME`
- etc.

So basically everything in `env.local.example`. Or should we leave variables that are prefixed with other component names (like `MAGPIE_ADMIN_USERNAME`) as they are?

Reply:
Everything that already has a component prefix such as `MAGPIE_ADMIN_USERNAME` can remain as is, since they are already avoiding conflict by being defined for that component. The generic ones such as `DATA_PERSIST_ROOT` can be improved. However, since generic ones are pretty much all used by override `env.local`, they need to support backward compatible names. The generic `SERVER_` prefixed variables could be switched to `BIRDHOUSE_` prefixes as well.